visualization/motion_planning_rviz_plugin/src/ui/motion_planning_rviz_plugin_frame.ui

```diff
@@ -628,7 +628,31 @@
              </widget>
             </item>
             <item>
-             <widget class="QDoubleSpinBox" name="exec_time_factor">
+             <widget class="QDoubleSpinBox" name="velocity_scaling_factor">
+              <property name="maximum">
+               <double>1.000000000000000</double>
+              </property>
+              <property name="singleStep">
+               <double>0.010000000000000</double>
+              </property>
+              <property name="value">
+               <double>1.000000000000000</double>
+              </property>
+             </widget>
+            </item>
+           </layout>
+          </item>
+          <item>
+           <layout class="QHBoxLayout" name="horizontalLayout_20">
+            <item>
+             <widget class="QLabel" name="label_20">
+              <property name="text">
+               <string>Acceleration Scaling:</string>
```

Comment:
Oh, I just saw your [screenshot](https://cloud.githubusercontent.com/assets/9295956/12263573/b2b92128-b8f5-11e5-9b36-884760dec9cf.png).

If you change this to "Accel Scaling" does it reduce the required screen width? The MoveIt widget is already too wide IMHO so I'd prefer not to do something to make it even wider